app/routes.js

```diff
@@ -7,7 +7,7 @@ import { logger } from './logger';
 import { withCatch, getHealthCheckDependencies, extractAccessToken } from './helpers/routerHelper';
 import { getDashboardContext } from './pages/dashboard/controller';
 import { getDescriptionContext, getDescriptionErrorContext, postOrPutDescription } from './pages/sections/description/controller';
-import { getSupplierSearchPageContext, validateSupplierSearchForm, getSupplierSearchPageErrorContext } from './pages/sections/supplier/search/controller';
+import * as supplierSearchController from './pages/sections/supplier/search/controller';
```

Comment:
how come you are importing like this?

Reply:
it wanted them all on separate lines since it was too long

Reply:
can change it just adds lot of lines of imports at the top and this routes file is just going it get even bigger.

Reply:
I think that is preferable to the massively long names when you call the functions??

Reply:
especially as it will probably push some of the lines over the char limit then will have to have multiple lines there instead